test/__mocks__/text.mock.ts

```diff
@@ -64,6 +64,7 @@ export const TRANSACTION_ERROR = "Transaction API request returned no response";
 export const CREATE_OE__MSG_ERROR = "Something went wrong creating Overseas Entity, transactionId =";
 export const FOUND_REDIRECT_TO = "Found. Redirecting to";
 export const REDIRECT_TO_SIGN_IN_PAGE = "User not authenticated, redirecting to sign in page, status_code=302";
+export const REDIRECT_TO_COMPANY_SIGN_IN_PAGE = "Found. Redirecting to test/signin?return_to=/update-an-overseas-entity/update-presenter&company_number=SA000392";
```

Comment:
What is the company_number on the query string? Should that be OE111129 instead?